swap_meet/item.py

```diff
@@ -23,7 +26,9 @@ def __str__(self):
         return f"An object of type {self.get_category()} with id {self.id}."
 
     def condition_description(self):
-        msg = CONDITIONS.get(self.condition)
-        return msg
-    
-
+        # If we are not allowed to use floor, I'll use int((self.condition + 0.5) // 1).
+        # The only downside is that for negative numbers, it rounds down (e.g., -2.4 becomes -3).
+        # Chose not to use round() because it has unexpected behavior ex 2.5 -> 2
+        condition = floor(self.condition + 0.5)
```

Comment:
👍 This will clamp a float condition to one of the values in your lookup table (assuming the valoues are in the expected range).

The use of `floor` does break our ban on other imports, but your alternative works fine for the range of expected values (we don't expect the condition to be negative). Note that the `// 1` (truncate divide by 1) isn't necessary as the `int()` call itself will  truncate the result (it's redundant).